### ISSUE TITLE
The latest built Python DLL (3.7.3 as of now) does not contain zlib module.

### ISSUE THREAD

**Issue by aliakseis** [microsoft/vcpkg#6499]
Importing zlib in the script:
    import zlib
causes error:
    bootstrap error "<class 'ModuleNotFoundError'>: No module named 'zlib':   

It works fine with just the Python37.DLL borrowed from the Python installation. Also it works with the older Python 3.6.x vcpkg build.

**Comment by PhoebeHui** [comment]
@aliakseis, it looks PR https://github.com/microsoft/vcpkg/pull/7051 fix similar issue, could you try the fix?

**Comment by aliakseis** [comment]
Tried the fix, it looks like it is not working yet. I did not try on something like a clean VM though.
I use Python 3.6 for now:
git checkout dfef7b111656e65a7e14078b6aaffa7f0a402308 -- ports/python3

**Comment by vicroms** [comment]
Fix in #7051 is working for me.

```
PS <vcpkg_root>\downloads\tools\python\python-3.7.3> python
Python 3.7.3 (v3.7.3:ef4ec6ed12, Mar 25 2019, 21:26:53) [MSC v.1916 32 bit (Intel)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import zlib
>>> exit()
```

**Comment by PhoebeHui** [comment]
@aliakseis, above PR has merged, I tried it, it also worked, please get latest vcpkg and have a try, reopen this if it still be an issue for you.
